src/test/java/org/takes/rq/multipart/RqMtFakeTest.java

```diff
@@ -203,8 +205,8 @@ public void closesAllParts() throws Exception {
             Assert.fail(exmessage);
         } catch (final IOException ex) {
             MatcherAssert.assertThat(
-                ex.getMessage(),
-                Matchers.containsString(closed)
+                ex,
```

Comment:
@victornoel We have lots of assertions in the same method, let's break this method in parts with just one assertion each.

Reply:
@paulodamaso I would be happy to add a todo but frankly I spent already too much time on this. The task wasn't really splitable because when upgrading qulice it's all or nothing, so I did it anyway. Refactoring tests is out of scope of the original issue.

Reply:
@victornoel All right, we hace lost a lot of time with this PR indeed. I'll create a new issue then